memcached/README.md

```diff
@@ -0,0 +1,45 @@
+# YCSB Memcached binding
+
+## Quick Start
+
+This section describes how to run YCSB on memcached.
+
+### 1. Install and start memcached service on the host(s)
+
+### 2. Install Java and Maven
+
```

Comment:
OK, happy to do that as a follow-up PR. Anything else you'd like to see as part of this PR, or is it good to go now?